uint8 motor_num
int16 pwm
---
int16 rpm
